src/Containers/Calendar/Components/CalendarDay/CalendarDay.tsx

```diff
@@ -0,0 +1,90 @@
+import moment from "moment";
+import React from "react";
+import { Text, StyleSheet } from "react-native";
+import { TransparentButton, TransparentButtonProps } from "src/Components/Buttons";
+import withTheme, { ThemeProps } from "src/Themes/withTheme";
+
+interface DefaultProps {
+  fontSize: number;
+  type: "month" | "week";
+}
+
+interface Props extends DefaultProps, ThemeProps, TransparentButtonProps {
+  currentDate: Date;
+  day: Date;
+  onDayPress(date: Date): void;
+}
+
+class CalendarDay extends React.PureComponent<Props> {
+  static defaultProps: DefaultProps = {
+    type: "month",
+    fontSize: 14,
+  };
+  public render() {
+    const { day, fontSize, type, ...rest } = this.props;
+    return (
+      <TransparentButton style={styles.dayContainer} onPress={this.handleDayPress} {...rest}>
+        {moment(day).isSame(this.props.currentDate, type) ? (
+          <Text
+            style={[
+              styles.day,
+              {
+                minWidth: fontSize * 2,
```

Comment:
This should be handled by theme framework through `getStyles` function. Styles should not be mixed with markup.